Documentation/ceph-cluster-crd.md

```diff
@@ -197,13 +195,13 @@ This allows to keep Rook components running when for example a node runs out of
 You can set resource requests/limits for Rook components through the [Resource Requirements/Limits](#resource-requirementslimits) structure in the following keys:
 
 - `mgr`: Set resource requests/limits for MGRs.
```

Comment:
The trend I see in most of the rest of the docs and the guideline I see most often is to make sure bulleted list items do *not* end in a period.